eras/conway/impl/src/Cardano/Ledger/Conway/Governance/Procedures.hs

```diff
@@ -343,12 +343,53 @@ instance EraPParams era => EncCBOR (ProposalProcedure era) where
         !> To pProcGovernanceAction
         !> To pProcAnchor
 
+data Committee era = Committee
+  { committeeMembers :: !(Map (Credential 'CommitteeColdKey (EraCrypto era)) EpochNo)
+  -- ^ Committee members with epoch number when each of them expires
+  , committeeQuorum :: !UnitInterval
+  -- ^ Quorum of the committee that is necessary for a successful vote
+  }
+  deriving (Eq, Show, Generic)
+
+instance Era era => NoThunks (Committee era)
+
+instance Era era => NFData (Committee era)
+
+instance Era era => DecCBOR (Committee era) where
+  decCBOR =
+    decode $
+      RecD Committee
+        <! From
+        <! From
+
+instance Era era => EncCBOR (Committee era) where
+  encCBOR Committee {committeeMembers, committeeQuorum} =
+    encode $
+      Rec (Committee @era)
+        !> To committeeMembers
+        !> To committeeQuorum
+
+instance EraPParams era => ToJSON (Committee era) where
+  toJSON = object . toCommitteePairs
+  toEncoding = pairs . mconcat . toCommitteePairs
+
+toCommitteePairs :: (KeyValue a, EraPParams era) => Committee era -> [a]
+toCommitteePairs committee@(Committee _ _) =
+  let Committee {..} = committee
+   in [ "members" .= committeeMembers
+      , "quorum" .= committeeQuorum
+      ]
+
 data GovernanceAction era
   = ParameterChange !(PParamsUpdate era)
   | HardForkInitiation !ProtVer
   | TreasuryWithdrawals !(Map (Credential 'Staking (EraCrypto era)) Coin)
   | NoConfidence
-  | NewCommittee !(Set (KeyHash 'Voting (EraCrypto era))) !Rational
+  | NewCommittee
+      -- | Old committee
+      !(Set (Credential 'CommitteeColdKey (EraCrypto era)))
```

Comment:
I may not have the complete context here, so I'm assuming we are not keeping `Committee era` here for the old committee because the old `UnitInterval` or the members' respective retirement `EpochNo`s are not relevant in the `GovernanceAction`.

Reply:
Yes, that is correct, as far as I understand it as well.